lib/particular-calendars/france.coutances.ts

```diff
@@ -0,0 +1,244 @@
+import { Precedences } from '../constants/precedences';
+import { CalendarDef } from '../models/calendar-def';
+import { Inputs } from '../types/calendar-def';
+
+import { France } from './france';
+
+// src:
+// - mr_fr_1982_ed2_coutances
+// - https://en.wikipedia.org/wiki/Roman_Catholic_Diocese_of_Coutances
+// - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese
+export class France_Coutances extends CalendarDef {
+  ParentCalendar = France;
+
+  inputs: Inputs = {
+    // src: mr_fr_1982_ed2_coutances
+    our_lady_of_pontmain: {
+      dateDef: { month: 1, date: 17 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    gaud_of_evreux_bishop: {
+      dateDef: { month: 1, date: 30 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    auguste_chapdelaine_priest: {
+      dateDef: { month: 2, date: 28 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    leon_de_carentan_bishop: {
+      dateDef: { month: 3, date: 1 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    placide_eulalie_victoire_jacqueline_viel_virgin: {
+      dateDef: { month: 3, date: 4 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marthe-le-bouteiller-vierge
+    marthe_aimee_adele_le_bouteiller_virgin: {
+      dateDef: { month: 3, date: 18 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    paterne_of_avranches_bishop_and_scubilion_abbot: {
+      dateDef: { month: 4, date: 16 },
+      martyrology: ['paterne_of_avranches_bishop', 'scubilion_abbot'],
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    william_firmatus_abbot: {
+      dateDef: { month: 4, date: 24 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    louis_grignion_de_montfort_priest: {},
+
+    // src: mr_fr_1982_ed2_coutances
+    achard_of_saint_victor_bishop: {
+      dateDef: { month: 4, date: 29 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    marculf_abbot: {
+      dateDef: { month: 5, date: 4 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    germanus_of_normandy_bishop: {
+      dateDef: { month: 5, date: 6 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marie-catherine-de-saint-augustin-vierge
+    catherine_of_saint_augustine_de_simon_de_longpre_virgin: {
```

Comment:
According [English Wikipedia](https://en.wikipedia.org/wiki/Catherine_of_St._Augustine) and [French Wikipedia](), her full:

- religious name is `Mary Catherine of Saint Augustine` (note: religious names seem to be localised, therefore in English `Marie` became `Mary` and the hyphen was removed in comparison with its French version);
- name is `Catherine de Simon de Longpré`;

therefore, the ID should be `mary_catherine_of_saint_augustine_catherine_de_simon_de_longpre_virgin`.

```suggestion
    mary_catherine_of_saint_augustine_catherine_de_simon_de_longpre_virgin: {
```